src/metaschema/oscal_metadata_metaschema.xml

```diff
@@ -93,19 +99,42 @@
          </remarks>
          <o:profile>
             <o:metadata>
-              <o:link rel="reference" href="#citation1">My citation</o:link>
+              <o:link rel="citation" href="#resource1">My citation</o:link>
```

Comment:
This doesn't appear at https://www.iana.org/assignments/link-relations/link-relations.xhtml (although the previous one didn't either).

Reply:
We use rels that are not in the link relations IANA registry. I think we need to come up with a means for the IANA relations and the OSCAL relations to co-exist.